packages/core/src/accessanalyzer/vue/constants.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export abstract class IamPolicyChecksConstants {
+    static readonly ValidatePolicySuccessWithFindings =
+        'Please view the problems panel to review the issues with your policy document. Policy checks should be run until issues are no longer found in your policy document.'
+    static readonly ValidatePolicySuccessNoFindings = 'Policy checks did not discover any problems with your policy.'
+    static readonly CustomCheckSuccessWithFindings =
+        'Result: FAIL. Please view the problems panel to review the issues with your policy document. Policy checks should be run until issues are no longer found in your policy document.'
+    static readonly CustomCheckSuccessNoFindings =
+        'Result: PASS. Policy checks did not discover any problems with your policy.'
+    static readonly CustomCheckFilePathSetting = 'aws.accessAnalyzer.policyChecks.customChecksFilePath'
+    static readonly CfnParameterFilePathSetting = 'aws.accessAnalyzer.policyChecks.cloudFormationParameterFilePath'
+    static readonly MissingReferenceDocError = 'Reference policy document is missing.'
```

Comment:
I don't think this is appropriate. We use the same for actions - the reference doc in that scenario is a not a policy document

Reply:
You're right, it should just be "reference document," will update.